### DISCUSSION TITLE
Response got Filtered - Doesn't make sense

### DISCUSSION THREAD

**Discussion by wrymn** [community/community#123081] [Copilot Conversations]
### Select Topic Area

Bug

### Body

<img width="532" alt="image" src="https://github.com/community/community/assets/12801017/ac8f5057-bcd9-47d2-a03f-28d7d9a14bb1">

Really? :D This is kinda broken, even referenced classes contain nothing "bad" that could get flagged

**Comment by Akash1134** [comment]
Hi @wrymn ,

Could you please review this [similar discussion](https://github.com/microsoft/vscode-copilot-release/issues/822#issuecomment-1924064375) and let us know if that helps?
